api-server/server/boot/certificate.js

```diff
@@ -68,6 +68,10 @@ const notCertifiedMessage = name => dedent`
   ${name} Certification
   `;
 
+const noChallengeMessage = name => dedent`
+  Oops! Something went wrong. Please try to claim the ${name} Certification again in a moment.
```

Comment:
While this might be transitory, it's probably worth adding the support email (or some other way of addressing it) in case people have tried a few times and it still fails.

Reply:
While @ahmadabdolsaheb and I were working on this we noticed that the Sentry error message overrides this one. In the end we removed this and just went with the Sentry report/error message.

Reply:
> we noticed that the Sentry error message overrides this one.

You mean the handled error message overides this one. I have happened to notice this too.

Lets create a fresh issue if there is not one.

Reply:
@raisedadead, yes, that's right. It would be good to show both like in the implementation here. I'll create a new issue if there isn't one already.